hashset_bool.go

```diff
@@ -0,0 +1,40 @@
+package hashset
+
+type Int64SetBool map[int64]bool
```

Comment:
please move `Int64SetBool` into internal package, we don't want to export it(for bench only).